File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/AuthenticationCommand.java
package com.happs.ximand.ringcontrol.model.object.command;

public class AuthenticationCommand extends BluetoothCommand {

    private static final byte AUTHENTICATE_COMMAND_CODE = 0;

    protected AuthenticationCommand() {
        super(AUTHENTICATE_COMMAND_CODE);
    }

    @Override
    public byte[] getCommand() {
        //return getCode() + ;
        return null;
    }
}

File: app/src/main/java/com/happs/ximand/ringcontrol/model/object/command/AuthenticationCommand.kt
package com.happs.ximand.ringcontrol.model.`object`.command

class AuthenticationCommand : BluetoothCommand<String?>(COMMAND_CODE) {
    override fun toByteArray(): ByteArray? {
        //return getCode() + ;
        return null
    }

    override fun getMainContent(): String {
        throw UnsupportedOperationException()
    }

    companion object {
        private const val COMMAND_CODE: Byte = 0
    }
}